cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,74 @@
 /// <reference types='cypress' />
-
+import { generateUser } from '../support/generate';
 describe('Student Registration page', () => {
   before(() => {
-
+    cy.visit('https://demoqa.com/automation-practice-form');
   });
+  const { email, firstName, lastName, address } = generateUser();
+  it('should enter vaild data to register', () => {
+    cy.visit('https://demoqa.com/automation-practice-form');
+
+    cy.get('#firstName').type(firstName);
+
+    cy.get('#lastName').type(lastName);
+
+    cy.get('#userEmail').type(email);
+
+    cy
+      .get('#genterWrapper > .col-md-9 > :nth-child(1) > .custom-control-label')
+      .click();
+
+    cy.get('#userNumber').type(1234567891);
```

Comment:
Magic number detected. It's a good practice to avoid magic numbers in your code. Consider declaring a constant with a meaningful name for this value.